src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -278,7 +315,7 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		this.codeActionList.value?.layout(height, maxWidth);
 
 		// List selection
-		if (this.viewItems.length < 1) {
+		if (this.viewItems.length < 1 || (this.viewItems.length === 1 && this.viewItems[0].isDocumentation)) {
```

Comment:
Can you instead check if every item is documentation instead of just checking the first? It is possible (but not likely) that multiple documentation items could be shown